### ISSUE TITLE
Pluralization of multiple variables within a single sentence?

### ISSUE THREAD

**Issue by klondikedragon** [Tienisto/flutter-fast-i18n#51]
Is it possible to have pluralization with multiple fields through linked translations? For example, if we have a `strings.i18n.yaml` like:
```
login:
  passwordInvalidFields:
    characters:
      one: $count character
      other: $count characters
    symbols:
      one: $count symbol
      other: $count symbols
  passwordInvalid: Password must be at least @:login.passwordInvalidFields.characters long with at least @:login.passwordInvalidFields.symbols
```

It will generate code for this, but then there is only one named count parameter: `t.login.passwordInvalid(count: _passwordMinLength)` so we can't specify the separate numbers for the number of characters and symbols. If instead the YAML is changed to:

```
login:
  passwordInvalidFields:
    characters:
      one: $numCharacters character
      other: $numCharacters characters
    symbols:
      one: $numSymbols symbol
      other: $numSymbols symbols
  passwordInvalid: Password must be at least @:login.passwordInvalidFields.characters long with at least @:login.passwordInvalidFields.symbols
```

then `t.login.passwordInvalid` will have the `numCharacters` and `numSymbols` parameters but then also have a `count` named parameter as well. Is it possible to use a named parameter other than `$count` for pluralization? What is the right way to do pluralization of multiple items within a single sentence?

**Comment by Tienisto** [comment]
Hello,

currently, it is not possible to have more than one pluralization in one "sentence".
So the best solution would be to combine these two translations manually with 3 calls (one for characters, one for symbols, and the last one for the combination).

But thanks for this issue. I will look into this. The problem is that the library does not know what the pluralization argument is, therefore I hardcoded it to `count`.

**Comment by Tienisto** [comment]
I just released `5.8.0` which includes this feature.

You need to add a parameter hint:
```yaml
login:
  passwordInvalidFields(numCharacters): # add param hint
    characters:
      one: $numCharacters character
      other: $numCharacters characters
    symbols(numSymbols): # add param hint
      one: $numSymbols symbol
      other: $numSymbols symbols
  passwordInvalid: Password must be at least @:login.passwordInvalidFields.characters long with at least @:login.passwordInvalidFields.symbols
```

**Comment by klondikedragon** [comment]
@Tienisto -- wonderful!!! thank you!
